.editorconfig

```diff
@@ -1,5 +1,5 @@
-# WARNING: Only edit this file in the Lombiq .NET Analyzers repository's folder. A copy of this file anywhere else will
-# be overwritten.
+# WARNING: Only edit this file in the Lombiq .NET Analyzers repository's "Lombiq.Analyzers" folder. A copy of this file
```

Comment:
You didn't do it, but still, the Build and Test workflow needs to be fixed, because it's necessary to check this too. It's not actually testing anyway (and it's something that shouldn't have gone past review).

So, please make this solution use the actual proper workflow: https://github.com/Lombiq/GitHub-Actions/blob/dev/.github/workflows/build-and-test-orchard-core.yml It'll also warn you that Lombiq.BaseTheem is not consolidated.

Reply:
Alrighty!

Reply:
This repo does not contain tests, so I added the build-dotnet.yml.

Reply:
Just call it "Build", then. Also, won't we have Node builds here, like SCSS compilation? Because then corepack should be enabled, what happens in the ebuild-and-test-orchard-core workflow but not in build-dotnet (by design).

Reply:
We can add that later when we will actually need it. Changed the name tho.